app/components/main_container_component.html.erb

```diff
@@ -3,7 +3,7 @@
   <main id="main-container" data-controller="google-cover-image" class="mb-5">
 
     <% # The quick report javascript requires this node to be on the page %>
-    <div class="flash_messages mx-auto my-3" style="max-width: 1000px;">
+    <div class="flash_messages<% if flash.any? %> mx-auto my-3<% end %>" style="max-width: 1000px;">
```

Comment:
Is there any reason to render the container at all?

Or, maybe the actual `AlertComponent` brings its own vertical margins, so they don't stack awkwardly?

Reply:
@cbeer According to the comment we are supposed to for the quick report javascript request the node to be on the page